test/README.md

```diff
@@ -1,5 +1,63 @@
 ## End-to-End Testing
 
+### AWS E2E Testing
+Featured in this directory is a `./awsE2ESolutionSetup` script whose goal is to encapsulate deploying an E2E environment in AWS. This script will perform the following key actions:
+1. Deploy an Elasticsearch source cluster to EC2 using the following [CDK](https://github.com/lewijacn/opensearch-cluster-cdk/tree/migration-es)
+   * The CDK context options for this CDK can be used to customize this deployment and can be provided with a custom context file using the `--context-file` option
+   * A sample CDK context file can be seen [here](defaultCDKContext.json)
+2. Build the Docker Images
+3. Deploy the Migration Assistant CDK (containing the Migration tooling and potentially an OpenSearch Domain) located in this repo [here](../deployment/cdk/opensearch-service-migration)
+   * The CDK context options for this CDK can be used to customize this deployment and can be provided with a custom context file using the `--context-file` option
+   * A sample CDK context file can be seen [here](defaultCDKContext.json)
```

Comment:
I suspect folks will find it confusing to use the same CDK Context file for both the source cluster setup and the Migration Assistant.  Well, at least, I'd find it confusing.  What do you think about having a separate file for each?  Should be a pretty fast change.

Reply:
Since each top-level object in this context file has an id that must be referenced if not using the default objects, I'm hoping that this won't be too confusing. I like being able to have multiple different configurations in one place and being able to select a given one to use.

Reply:
I think what feels "off" to me isn't that there's multiple, different config "objects" in the files, it's that there's multiple types of config "objects" in the file.  It seems like we're already exposing users to the concept that there's two different CDK entities they're using by having them specify keys into this config JSON file.  Having a separate config file seems like a natural extension, and will enable us to do things like validate all the entries in each file using a common schema.

How much work do you think it would be to just split them out?

Reply:
Personally I feel its an extra tax on a user to now have to create two files to use this script, and would expect any validation of context options to be done by the relevant CDK.

That being said, its not something high on my list of concerns and the change itself should be small and then require updating our Jenkins pipelines. Will see about adding this into the PR

Reply:
Added now 👍